/* Automatically generated by po2tbl.sed from pybliographer.pot.  */

#if HAVE_CONFIG_H
# include <config.h>
#endif

#include "libgettext.h"

const struct _msg_ent _msg_tbl[] = {
  {"", 1},
  {"This is %s, version %s", 2},
  {"This is free software with ABSOLUTELY NO WARRANTY.", 3},
  {"For details, type `warranty'.", 4},
  {"%s: error: can't open file `%s'", 5},
  {"For help, run %s and type `help' at the prompt", 6},
  {"\
Useful commands:\n\
\thelp     to get some help\n\
\tquit     to quit\n", 7},
  {"entry has no key `%s'", 8},
  {"error: field `%s' appears more than once in the definition of `%s'", 9},
  {"no update method defined", 10},
  {"database `%s' is unknown in this Reference", 11},
  {"can't update a database of type Reference ()", 12},
  {"key `%s' is malformed", 13},
  {"%s:%d: key `%s' already defined", 14},
  {"in entry %s: nested crossrefs", 15},
  {"Native editing of field `%s'", 16},
  {"Edit the field in its Native format", 17},
  {"Drag an Entry to define a Cross-Reference", 18},
  {"Remove the current Cross-Reference", 19},
  {"Add Extra Field", 20},
  {"Cross-Reference `%s' is unexistent in the database", 21},
  {"Mandatory", 22},
  {"Optional", 23},
  {"Extra", 24},
  {"Invalid field name `%s'", 25},
  {"Key `%s' already defined", 26},
  {"Invalid identifier `%s'", 27},
  {"Identifier `%s' already exists", 28},
  {"File", 29},
  {"Bibliography type:", 30},
  {"Format", 31},
  {"Formatting Style:", 32},
  {"Output type:", 33},
  {"New", 34},
  {"Open", 35},
  {"Save", 36},
  {"Save As...", 37},
  {"Previous Documents", 38},
  {"Quit", 39},
  {"About...", 40},
  {"Submit a Bug Report", 41},
  {"Copy", 42},
  {"Cut", 43},
  {"Paste", 44},
  {"New...", 45},
  {"Edit...", 46},
  {"Delete", 47},
  {"Search...", 48},
  {"Sort by Key", 49},
  {"Sort by Type", 50},
  {"Cite in LyX", 51},
  {"Format As...", 52},
  {"Preferences...", 53},
  {"Edit", 54},
  {"Configure", 55},
  {"Formatting", 56},
  {"Help", 57},
  {"Search", 58},
  {"Cite", 59},
  {"Exit", 60},
  {"Welcome to Pybliographic", 61},
  {"Modified database `%s' [%s, %d]", 62},
  {"Database `%s' [%s, %d]", 63},
  {"New file", 64},
  {"Database is modified.\n", 65},
  {"Discard changes ?", 66},
  {"Quit without saving ?", 67},
  {"can't save file: %s", 68},
  {"Save As", 69},
  {"\
File `%s' exists.\n\
Overwrite it ?", 70},
  {"\
Can't save file:\n\
%s", 71},
  {"Not yet implemented...", 72},
  {"\
Please, create a new database first.\n\
Use File/New for example", 73},
  {"editing is disabled in this database", 74},
  {"edition is disabled in this database", 75},
  {"\
Can't connect to LyX:\n\
%s", 76},
  {"Really remove entry %s ?", 77},
  {"\
Please install bug-buddy\n\
to use this feature", 78},
  {"This program is copyrighted under the GNU GPL", 79},
  {"Gnome interface to the Pybliographer system.", 80},
  {"\
Submit a bug report\n\
 using bug-buddy", 81},
  {"Pybliographer Home Page", 82},
  {"Please open a database first", 83},
  {"Bibliography", 84},
  {"Field", 85},
  {"Pattern", 86},
  {"Simple Search", 87},
  {"Search command:", 88},
  {"Expert Search", 89},
  {"Full database", 90},
  {"Ready to search !", 91},
  {"internal error during evaluation", 92},
  {"\
while compiling %s\n\
error: %s", 93},
  {"found %d matches", 94},
  {"%d matches in this node", 95},
  {"\
The following errors occured:\n\
\n", 96},
  {"warning: can't import %s: %s", 97},
};

int _msg_tbl_length = 97;
